.circleci/config.yml

```diff
@@ -759,35 +759,59 @@ jobs:
       - run:
           name: Clippy linting on burner
           working_directory: ~/project/contracts/burner
-          command: cargo clippy -- -D warnings
+          command: |
+            mkdir -p target/wasm32-unknown-unknown/release
+            touch target/wasm32-unknown-unknown/release/burner.wasm
+            cargo clippy --tests -- -D warnings
```

Comment:
Really? I wondered why integration test hints weren't being reported, and now I know why.

Reply:
Yeah, same is `cargo check`, `cargo build`. I was not aware of this too.

Reply:
This seems to be fixed in `nightly` (`cargo clippy -Z unstable-options`). See https://github.com/rust-lang/rust-clippy/issues/1495.